app/src/FreeDeck.cpp

```diff
@@ -30,6 +30,10 @@ byte addressToScreen[] = ADDRESS_TO_SCREEN;
 byte addressToButton[] = ADDRESS_TO_BUTTON;
 #endif
 
+unsigned long startTime;
+unsigned long currentTime;
```

Comment:
make this a local variable in each function (`checkTimeOut` and `switchOnScreens`)